test/cases/all/values.yaml

```diff
@@ -6,6 +6,7 @@ config:
   sessions_secret: 4484766F-5016-4077-B8E0-0DE1D637854B
   server_url: http://bindplane.local:3001
   remote_url: ws://bindplane.local:3001
+  licenseUseSecret: true
```

Comment:
This test case will mount the license from the secret created in the CI workflow.